.travis.yml

```diff
@@ -93,9 +97,6 @@ matrix:
     - { rvm: *jruby_head, jdk: *jdk_8, env: *rails_master_jruby }
 
   exclude:
-    - { rvm: *ruby_2_4,  env: *rails_4_1 }
-    - { rvm: *ruby_2_5,  env: *rails_4_1 }
-    - { rvm: *ruby_2_6,  env: *rails_4_1 }
```

Comment:
curious to know, you might have reasons to remove these?

Reply:
I don't think they add much incremental value (I changed them from opt-out to opt-in)

Reply:
pushed again. diff was weird

Reply:
looks like these lines are still missing from the new version. 

without these lines under the `exclude` section, default build matrix will try to run tests with these env and fail, eventually resulting in failure on travis